android/src/main/java/com/github/wuxudong/rncharts/markers/RNImageMarker.java

```diff
@@ -0,0 +1,173 @@
+package com.github.wuxudong.rncharts.markers;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Rect;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
+
+import com.github.mikephil.charting.charts.Chart;
+import com.github.mikephil.charting.components.IMarker;
+import com.github.mikephil.charting.data.Entry;
+import com.github.mikephil.charting.highlight.Highlight;
+import com.github.mikephil.charting.utils.FSize;
+import com.github.mikephil.charting.utils.MPPointF;
+
+import java.lang.ref.WeakReference;
+import java.util.Map;
+
+/**
+ * Extends com.github.mikephil.charting.components.MarkerImage to allow
```

Comment:
if we ever want to merge our changes upstream, would it be cleaner to copy `MarkerImage` into this repo, and create our own `RefreshableMarkerImage` that extends the MarkerImage and overrides the `refreshContent` method as necessary? That way it's also easier to spot the difference from the original MarkerImage source code if the MPAndroid library updates it.

Reply:
actually, I take that back! I think it's fine as is, and can just add comment in code that only `refreshContent` is changed!